implementations/c/ockam/vault/atecc608a/atecc608a.c

```diff
@@ -93,7 +93,7 @@ typedef struct {            /*!< Byte(s): Description
  * @brief EEPROM slot configuration data
  */
 typedef struct {
-  ockam_vault_secret_t* secret;
+  bool                  taken;
```

Comment:
what does `taken` mean?

Reply:
Previously ATECC vault stored pointers to all secrets, but it lead to memory management problems + it wasn't really necessary functionality. So, instead I replaced secret pointer with bool value that just indicates that given slot is used for some secret.